pkg/generator/caches.go

```diff
@@ -92,8 +91,8 @@ func (caches *Caches) validateIngress(translatedIngress *translatedIngress) erro
 	return nil
 }
 
-func (caches *Caches) addTranslatedIngress(ingress *v1alpha1.Ingress, translatedIngress *translatedIngress) error {
-	caches.logger.Debugf("adding ingress: %s/%s", ingress.Name, ingress.Namespace)
+func (caches *Caches) addTranslatedIngress(translatedIngress *translatedIngress) error {
+	caches.logger.Debugf("adding ingress: %s/%s", translatedIngress.name.Name, translatedIngress.name.Namespace)
```

Comment:
More related to the previous PR than this one, but maybe we should add a `String()` function in `NamespacedName`?

Reply:
oops, I could've used this here. It has that. Lemme do a pass to turn all namespace/name usages into the right direction too :joy: